classes/StripeIdempotencyKey.php

```diff
@@ -125,23 +125,21 @@ public function getByIdPaymentIntent($id_payment_intent)
      * @throws PrestaShopException
      * @throws \Stripe\Exception\ApiErrorException
      */
-    public function createNewOne($id_cart, $datasIntent)
+    public function createNewOne($id_cart, $intentData)
```

Comment:
on a minor version, in a public method you should not change a name of a parameter. It could be override by someone and introduce a BC.